.env.example

```diff
@@ -3,7 +3,8 @@
 # OPENAI_API_KEY=
 OPENAI_CHAT_MODEL=gpt-4
 PORT=3000
-DB_USER=
-DB_PASSWORD=
-DB_NAME=
-DB_PORT=
+PGUSER=
+PGHOST=
+PGPASSWORD=
+PGDATABASE=
+PGPORT=
```

Comment:
There are the default values so we could use just `new Pool()` or `new Client()` instead of ```
new Pool({
      user: process.env.DB_USER,
      host: 'localhost',
      database: process.env.DB_NAME,
      password: process.env.DB_PASSWORD,
      port: process.env.DB_PORT,
    });
```